src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationExtensions.cs

```diff
@@ -39,7 +61,7 @@ public static class AzureAppConfigurationExtensions
             Action<AzureAppConfigurationOptions> action,
             bool optional = false)
         {
-            return configurationBuilder.Add(new AzureAppConfigurationSource(action, optional));
+            return configurationBuilder.Add(new AzureAppConfigurationSource(action, optional: optional));
```

Comment:
>optional: optional [](http://example.com/codeflow?start=84&length=18)

Are we adopting some new coding practice? "optional: optional" doesn't make code easier to read, IMO.

Reply:
I only added this because it was would be an error otherwise when there were default values for environmentName and optional parameters. Removing both of those default values so I'll remove this too.

Reply:
Cool. We have one more reason to remove the default values :)